src/main/java/com/futurcollars/lesson7/zad2/Movie.java

```diff
@@ -0,0 +1,34 @@
+package com.futurcollars.lesson7.zad2;
+
+public class Movie {
+
+    private String titleOfMovie;
+    private String fullNameDirector;
+    private int movieTime;
+    private int yearOfProductionMovie;
```

Comment:
Zauważ, że są to atrybuty w klasie Movie, więc jak masz tytuł to nie musisz go nazywać "tytuł filmu" wiadomo, że to dotyczy tego obiektu :) dlatego wystarczy jak nazwiesz te pola: title, directorName, duration i yearOfProduction albo yearOfRelease

Reply:
pomyśl też o oznaczeniu słowem final tych pól, które nie będą mogły się zmieniać :)